README.md

```diff
@@ -50,16 +52,26 @@ Install the firebase cli tools with the following commands:
 
 ```bash
 npm install -g firebase-tools
-firebase login  # Login to the Google account destined to manage the Firebase project
+firebase login  # Login to an account that has admin rights for the Firebase project
+```
+
+### Configuration
+
+Create the configuration files for a default firebase project:
+
+```bash
+cp .firebaserc.example .firebaserc
+cp webapp/src/lib/api/firebase-config.example.ts webapp/src/lib/api/firebase-config.ts
 ```
 
+This should be enough for local development with emulators. Be sure to input the correct project id and keys once you actually deploy the project to firebase!
```

Comment:
Clarify a little more: be sure to update the project ID in the .firebaserc file and keys in ....

Reply:
fixed in 9a95290e5d04a9f824d3501641fb11e7b25e52ff